src/main/webapp/app/entities/flair-bi/filter/header-pin-filter.service.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('flairbiApp')
+        .factory('headerPinFilterService', headerPinFilterService);
+
+    headerPinFilterService.$inject = ['$http'];
```

Comment:
this service is not required...can you place the http call in features service

Reply:
done